mmyolo/models/necks/yolov5_pafpn.py

```diff
@@ -156,7 +156,7 @@ def build_bottom_up_layer(self, idx: int) -> nn.Module:
         """
         return CSPLayer(
             make_divisible(self.in_channels[idx] * 2, self.widen_factor),
```

Comment:
It should be optimized if you refer to the yolov6 neck method because the current writing method must satisfy a multiple of 2 relationships and cannot support any number of channels.